CHANGELOG.md

```diff
@@ -4,15 +4,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.0.0]
```

Comment:
The build output isn't 1.0.0. It's currently still 0.5.0 and needs to be changed to 0.6.0.

Reply:
since it's a major change should it not be 1.0.0?